.github/workflows/ci.yml

```diff
@@ -0,0 +1,91 @@
+name: Run Integration Tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - project: tests/Aspire.Hosting.Elasticsearch.Tests/Aspire.Hosting.Elasticsearch.Tests.csproj
+            name: Elasticsearch
+            filter: "WithDataShouldPersistStateBetweenUsages"
+          - project: tests/Aspire.Hosting.PostgreSQL.Tests/Aspire.Hosting.PostgreSQL.Tests.csproj
+            name: PostgreSQL
+            filter: "Aspire.Hosting"
```

Comment:
Are these filters just for now? I wouldn't expect us to use them when we did this for real.

Reply:
I was playing with how to enable filters. I don't really want any in the final version.